poetry.lock

```diff
@@ -1,44 +1,42 @@
-# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
Did this lock file need to be updated? I see no corresponding change in pyproject.toml. Unfortunately, wholesale updates to the lockfile can cause unexpected breaks downstream.

Reply:
no, not really

Reply:
Can we reset it to match whats on `master`?